src/Components/Nav/Nav.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './nav.css';
+
+const Nav = () => {
+    return(
+        <nav className="nav">
+            <ul className="nav__list">
+                <li className="nav__item"><a href="" className="nav__link">List of tasks</a></li>

```

Comment:
`className="nav__link"` <-- Don't use string as a class name in this place. You can easy make typo in long classnames and it can be difficult to find.

 I suggest use something like `className={classes.nav__link}`
 benefits:
- IDE will suggest you properly class names
- if you make typo in classname you will see an error with the message that help you fast fix the issue.